routes/users.js

```diff
@@ -113,7 +127,7 @@ router.post('/signup', csrfProtection, validateSignup, handleValidationErrors, a
 
   const validatorErrors = validationResult(req);
   if (validatorErrors.isEmpty()) {
-    const user = db.User.build({
+    const user = await db.User.build({
```

Comment:
`.build` is not an asynchronous operation. Usually you will do `.build` to create an instance in your server program, and then `.save` to commit that instance as a row into your database. `.save` WILL be asynchronous, and should be used with the `await` operator; using `await` with `.build` doesn't actually do anything.